### DISCUSSION TITLE
Art of the Octocat book

### DISCUSSION THREAD

**Discussion by ChrisArchitect** [community/events#68] [Keynotes]
Caught my eye and thought some might be interested in this swag item that was exclusive on-site:

**Coming Soon**

### Art of The Octocat📘

> If you are attending [#githubuniverse2022](https://twitter.com/hashtag/githubuniverse2022) You might stumble on a "Art of The Octocat" book.  
> It is a selection of all the amazing work done by all the artists that work and have worked at [@GitHub](https://twitter.com/github). ![pic.twitter.com/ZYw8WYfZxn](https://pbs.twimg.com/media/FhKmYJ3VUAA5mD1?format=jpg&name=medium)
> 
> I'd also like to acknowledge the amazing work done by everyone at [@VolumeSF](https://twitter.com/VolumeSF) .  
> It was great to see the process of it being put together! 
> ![pic.twitter.com/9CSsi4VihW](https://pbs.twimg.com/media/FhKnM6rUoAIDelo?format=jpg&name=small)
> ![](https://pbs.twimg.com/media/FhKnPUDVsAUW8TJ?format=jpg&name=small)
> ![](https://pbs.twimg.com/media/FhKnbfWUoAAR8uy?format=jpg&name=small)
> 
> If you didn't get the chance to pick up one of these, they are going to be available sometime December. [#artoftheoctocat](https://twitter.com/hashtag/artoftheoctocat) ![pic.twitter.com/q65lm8Cq9h](https://pbs.twimg.com/media/FhKnX4fVUAEtoJR?format=jpg&name=medium)
> 
> — João Ribeiro [November 10, 2022](https://twitter.com/Wodzgn/status/1590518660313251841)

Props to João for all the info. Hotlinked all the images ¯\_(ツ)_/¯

Happy Universe!

(mods delete this if not welcome content/more official announcements coming later, thx!)

**Comment by jricciardi** [comment]
A totally welcome post, @ChrisArchitect! Thank you for sharing for anyone who couldn't attend in person ❤️
